WordPress/Classes/Services/SuggestionService.swift

```diff
@@ -0,0 +1,118 @@
+import Foundation
+
+extension NSNotification.Name {
+    static let suggestionListUpdated = NSNotification.Name("SuggestionListUpdated")
+}
+
+@objc extension NSNotification {
+    public static let suggestionListUpdated = NSNotification.Name.suggestionListUpdated
+}
+
+@objc public class SuggestionService: NSObject {
```

Comment:
It would be good to add a comment here explaining what the class is used for. We have a number of different suggestion-related classes in the app (like domain suggestions, Reader suggestions), so would be helpful to differentiate.